engine/src/canvas.lcb

```diff
@@ -4565,9 +4567,14 @@ end syntax
 Summary:	Begin a new drawing layer on a canvas.
 
 mCanvas:		An expression which evaluates to a canvas.
+mBounds:		An expression which evaluates to a rectangle.
 mEffect:	An expression which evaluates to an effect.
 
-Description:	Begins a new layer on <mCanvas> to group drawing operations for which the opacity, blend mode, and optional effect are to be applied to as a whole rather than individually.
+Description:	Begins a new layer on <mCanvas> to group drawing operations for
```

Comment:
@livecodeian This has the same issue as mentioned on the property set patch. Also probably good to have examples for each the two new ones.